joomla/joomla-cms
# [4.0] Frontend article submit permission

Pull Request for Issue #32117.

### Summary of Changes
This PR improves frontend article submission form so that the form will respect permission of the pre-selected category (which is selected in the menu item parameter). See https://github.com/joomla/joomla-cms/issues/32117 to understand the issue better

### Testing Instructions
1. Create a new user account, assign the account to **Author** user group.
2. Edit an existing article category (or create a new category), in the Permissions tab, make sure **Edit State** permission for **Author** user group is set to Allowed.
3. Create a menu item to link to **Create Article** menu option.
4. Login to frontend using the user account which belong to Author user group above, access to the menu item to try to submit article:
- Before patch: You do not have Edit State permission, so you cannot change settings such as Published, Start Publishing, Start Featured....
- After patch: You can select status for the article (for example Published), select date For Start Publishing, Start Featured.. (in Publish tab)
